Phonebook.Frontend/src/app/app.module.ts

```diff
@@ -53,6 +53,7 @@ import { environment } from 'src/environments/environment';
 import { FloorplanService } from './services/floorplan.service';
 import { SearchComponent } from './shared/components/search/search.component';
 import { HttpRedirectToLogin } from 'src/app/shared/interceptors/HttpRedirectToLogin';
+import { FormsModule } from '@angular/forms';
```

Comment:
Whats the form module for?

Reply:
It's needed for the ngModel property in mat-select in the dashboard.component.html file.

Reply:
k